evm/deploy-amplifier-gateway.js

```diff
@@ -240,6 +240,7 @@ async function deploy(config, chain, options) {
         printInfo('Proxy deployment args', `${implementation.address}, ${params}`);
 
         contractConfig.operator = operator;
+        contractConfig.setupParams = params;
```

Comment:
```suggestion
        const proxyDeploymentArgs = [implementation.address, owner, params];
        contractConfig.proxyDeploymentArgs = proxyDeploymentArgs;
```
this handles the full source verification requirement

Reply:
reuse `proxyDeploymentArgs` below as well